docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -213,6 +215,60 @@ inside the container, so you can simply run:
 
 5. Stop and remove container
 
+.. code-block:: bash
+
+    docker stop pelux-build
+    docker rm pelux-build
```

Comment:
Why do any want to remove the container? Explain when this can be relevant.

Reply:
Well, if we're stopping it why would we delete it

Reply:
My mistake, I thought it was the image pelux which was built that was deleted, not the contianer pelux-build.